### PR TITLE
Fix link to Akka Projection SourceProvider API

### PR DESCRIPTION
**Pull request by renatocaval**

### CODE REVIEW

File: blog/_posts/2020-09-10-akka-projection-1.0.0.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ As the source you can select from:
 
 * [Events from Akka Persistence](https://doc.akka.io/docs/akka-projection/current/eventsourced.html)
 * [Messages from Kafka](https://doc.akka.io/docs/akka-projection/current/kafka.html)
-* Building your own [SourceProvider](doc.akka.io/api/akka-projection/1.0.0/akka/projection/scaladsl/SourceProvider.html)
+* Building your own [SourceProvider](https://doc.akka.io/api/akka-projection/1.0.0/akka/projection/scaladsl/SourceProvider.html)
```
Comment (patriknw) [inline blog/_posts/2020-09-10-akka-projection-1.0.0.md:23]: shouldn't we use `current` in those links?

Comment (renatocaval) [inline blog/_posts/2020-09-10-akka-projection-1.0.0.md:23]: If are sure that the API is stable, yes. 

On the other hand, we are already using `current` to link to the documentation page. So the risk of broken links already exist.

Comment (renatocaval) [reply blog/_posts/2020-09-10-akka-projection-1.0.0.md:23]:
```suggestion
* Building your own [SourceProvider](https://doc.akka.io/api/akka-projection/current/akka/projection/scaladsl/SourceProvider.html)
```

Comment (renatocaval) [reply blog/_posts/2020-09-10-akka-projection-1.0.0.md]: Done

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**